CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 1.50.1
+
+### New features
+
+* Enabled `:set-timezone` ([docs](https://www.metabase.com/docs/latest/configuring-metabase/localization#report-timezone)) Metabase feature in the driver. ([#200](https://github.com/ClickHouse/metabase-clickhouse-driver/issues/200))
+
+### Other
+
+* The driver now uses [`session_timezone` ClickHouse setting](https://clickhouse.com/docs/en/operations/settings/settings#session_timezone). This is necessary to support the `:set-timezone` feature; however, this setting [was introduced in 23.6](https://clickhouse.com/docs/en/whats-new/changelog/2023#236), which makes it the minimal required ClickHouse version to work with the driver.
```

Comment:
>  The setting takes precedence over the globally configured (server-level) implicit time zone. 

I hope every user reads this warning. Should we add it to the changelog?

Reply:
I think I will add some additional explanation when I will rework the README and the website docs.